app/code/Magento/Inventory/Block/Adminhtml/Edit/Button/AbstractButton.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace  Magento\Inventory\Block\Adminhtml\Edit\Button;
+
+use Magento\Backend\Block\Widget\Context;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\InventoryApi\Api\SourceRepositoryInterface;
+
+/**
+ * Class AbstractButton
+ */
+abstract class AbstractButton
```

Comment:
Try to avoid usage of abstract classes in such cases.
We do not encourage usage and introduction of the new inheritance based APIs